lib/renderer/content-scripts-injector.js

```diff
@@ -1,6 +1,8 @@
-const {ipcRenderer} = require('electron')
+const {ipcRenderer, webFrame} = require('electron')
 const {runInThisContext} = require('vm')
 
+webFrame.registerURLSchemeAsPrivileged('chrome-extension')
```

Comment:
I'm wondering if this should only be done if content scripts are actually being injected instead of all the time even when none exist.

Maybe it could be run from inside `injectContentScript`?